common/sgx/tlsverifier.c

```diff
@@ -172,3 +202,136 @@ oe_result_t oe_verify_attestation_certificate(
     oe_free(report);
     return result;
 }
+
+/**
+ * oe_verify_attestation_certificate_with_evidence
+ *
+ * This function performs a custom validation on the input certificate. This
+ * validation includes extracting attestation evidence extension from the
+ * certificate before validating this evidence. An optional
+ * claim_verify_callback could be passed in for a calling client to further
+ * validate the claims of the enclave creating the certificate.
+ * @param[in] cert_in_der a pointer to buffer holding certificate contents
+ *  in DER format
+ * @param[in] cert_in_der_len size of certificate buffer above
+ * @param[in] claim_verify_callback callback routine for custom claim checking
+ * @param[in] arg an optional context pointer argument specified by the caller
+ * when setting callback
+ * @retval OE_OK on a successful validation
+ * @retval OE_VERIFY_FAILED on quote failure
+ * @retval OE_INVALID_PARAMETER At least one parameter is invalid
+ * @retval OE_FAILURE general failure
+ * @retval other appropriate error code
+ */
+oe_result_t oe_verify_attestation_certificate_with_evidence(
+    uint8_t* cert_in_der,
+    size_t cert_in_der_len,
+    oe_verify_claims_callback_t claim_verify_callback,
+    void* arg)
+{
+    oe_result_t result = OE_FAILURE;
+    oe_cert_t cert = {0};
+    uint8_t* report = NULL;
+    size_t report_size = 0;
+    oe_report_header_t* header = NULL;
+    uint8_t* pub_key_buf = NULL;
+    size_t pub_key_buf_size = KEY_BUFF_SIZE;
```

Comment:
"buf" vs "BUFF" is similarly inconsistent in terms of # of F's

Reply:
made it two Fs